magefiles/test.go

```diff
@@ -8,33 +8,21 @@ import (
 	"os"
 	"strings"
 
-	"github.com/authzed/ctxkey"
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
 )
 
 type Test mg.Namespace
 
-var (
-	emptyEnv map[string]string
-	ctxMyKey = ctxkey.New[bool]()
-)
+var emptyEnv map[string]string
 
 // All Runs all test suites
-func (t Test) All(ctx context.Context) error {
+func (t Test) All(ctx context.Context) {
 	ds := Testds{}
 	c := Testcons{}
-	cover := parseCommandLineFlags()
-	ctx = ctxMyKey.WithValue(ctx, cover)
 	mg.CtxDeps(ctx, t.Unit, t.Integration, t.Steelthread, t.Image, t.Analyzers,
 		ds.Crdb, ds.Postgres, ds.Spanner, ds.Mysql,
 		c.Crdb, c.Spanner, c.Postgres, c.Mysql)
-
-	if !cover {
```

Comment:
Presumably, nobody cares about `mage test:all` running coverage or not. I assume that we can leave coverage collection to CI/CD